api/app/Http/Controllers/UserController.php

```diff
@@ -116,6 +116,7 @@ public function putOne($id, Request $request)
 
         $model = $this->repository->getNewModel();
         $model->fill($request->all());
+        $model->email = $request->get('email');
```

Comment:
Why is this line needed? Doesn't the `->fill()` method assign the requested email to the model? If the request doesn't contain email this may throw an error?